CONTRIBUTING.rst

```diff
@@ -0,0 +1,39 @@
+Contribution Guidelines
+#######################
+
+As an open-source project, we welcome and encourage the community to submit
+patches directly to the project.  In our collaborative open source environment,
+standards and methods for submitting changes help reduce the chaos that can result
+from an active development community.
+
+This document briefly summarizes the full `Contribution
+Guidelines <http://projectacrn.github.io/latest/developer-guides/contribute_guidelines.html>`_
+documentation.
+
+* ACRN uses the permissive open source `BSD 3-Clause license`_
+  that allows you to freely use, modify, distribute and sell your own products
+  that include such licensed software.
+
+* There are some imported or reused components of the ACRN project that
+  use other licensing and are clearly identified.
+
+* The Developer Certificate of Origin (DCO) process is followed to
+  ensure developers are following licensing criteria for their
+  contributions, and documented with a ``Signed-off-by`` line in commits.
+
+* ACRN development workflow is supported on Linux.
+
+* Source code for the project is maintained in the GitHub repo:
+  https://github.com/projectacrn/acrn-hypervisor
+
+* Issue and feature tracking is done using GitHub issues in this repo.
+
+* A Continuous Integration (CI) system runs on every Pull Request (PR)
+  to verify several aspects of the PR including Git commit formatting,
+  Coding Style, sanity checks builds, and documentation builds.
```

Comment:
No reason to capitalize "coding style". Also: "sanity checks builds" → "sanity-check builds".